playbooks/documentation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Documentation
+description: How and where to document or share content
+---
+
+# Documentation
+
+As engineers we operate in many parallel forums: GitHub, Jira, Slack, Google docs, e-mail, Notion, and so on. What
+content should go where?
+
+## Documentation or discussion of a specific system
+
+When documentation relates to a particular system, leverage in-repository `README`s or `doc/`s wherever possible.
+This includes when the consumers may not exclusively be engineers.
+
+E.g.:
+
+- How to set-up a local development environment
+- Definitions of domain models
+- API documentation
+
+It _almost_ goes without saying, but other repo-specific interactions such as presenting the rationale for a design
+or offering code-review should leverage the built-in mechanisms of
+[commit messages](/playbooks/engineer-workflow.md#commits),
+[pull requests](/playbooks/engineer-workflow.md#pull-requests), and comments.
+
+## Public versus private
+
+When creating a repo, contributing a doc, or discussing in-progress work, consider whether it's appropriate for
+this content to be public or private.
+
+Content that should always be private:
+
+- Contact or role details for individuals
+- Information about candidates or new hires before they've joined or consented to announce their move
+- Production data or logs (including indirectly through migration output, etc.)
+- Repositories representing substantial business value, novelty, or criticality (such as the main API, content
+  management system, or auction operator tools)
+
+Content that _maybe_ should be private, depending on its sensitivity:
+
+- Security or compliance topics, including pending security work items or vulnerability-handling procedures
```

Comment:
I strongly disagree here. I think compliance and security information should always be private by default, and deliberately made public when appropriate.

This is aligned with the Priciple of Least Privilege - a well known and widely accepted security principle.

Reply:
Interesting! I think I got the opposite feedback about other forums, like the #security slack channel and trello board, which were invite-only. (In those cases we were debating between being public to the whole Artsy org and _not_ the whole world, but similar principles apply.)

I guess I meant to allow some space for more general security discussion without shunting them out of the repo to which they're relevant. E.g., if we were considering adding [subresource integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to asset references in Force, could that be discussed on a PR there?

Reply:
I think there's a big difference between having security information being open within a private organization by default versus being open to the public by default, but point well taken. 

I do think it's generally safer to keep all security matters closed by default. In particular, I think it would be undesirable from a business perspective to have security incidents open as they progress. I would prefer to be less ambiguous, make a clear statement that, by default, these things should not be in the public-domain, than selectively promote such content to be public.

Reply:
Agreed. Moving this to the other section.